.github/workflows/test-ros2.yml

```diff
@@ -17,12 +17,15 @@ jobs:
     
     strategy:      
       matrix:
-        distro: ['dashing']
+        distro: ['dashing','foxy']
         gazebo: [9]
         include:
         - distro: dashing
           gazebo: 9
-          ubuntu_distro: bionic      
+          ubuntu_distro: bionic
+        - distro: foxy
+          gazebo: 9
```

Comment:
should be 11? https://www.ros.org/reps/rep-2000.html#foxy-fitzroy-may-2020-may-2023

Reply:
Good catch.